.travis.yml

```diff
@@ -3,8 +3,12 @@ cache:
     - $HOME/.m2
 language: java
 jdk:
-  - openjdk7
   - oraclejdk8
+  - oraclejdk9
+  - oraclejdk10
+  - oraclejdk11
+before_install:
+  - chmod +x mvnw
```

Comment:
I don't think this is necessary.

Reply:
For some reason the build fails for me, without that line.

Reply:
https://travis-ci.org/timgifford/maven-buildtime-extension/builds/395315124

Reply:
@mithuns When looking at the full diff for this PR, Github shows:
>  mvnw 100755 → 100644

If you undo that permission change, this `chmod +x mvnw` won't be necessary.

Reply:
Oh yeah, I see it now. Reverted to the original file mod permissions. 
Thanks